trading_ig/rest.py

```diff
@@ -866,8 +865,8 @@ def fetch_open_positions(self, session=None, version='2'):
 
         if self.return_dataframe:
 
-            list = data["positions"]
-            data = pd.DataFrame(list)
```

Comment:
What is the purpose of this change?

Reply:
Nothing fundamental, just that it is typically considered bad practice to name variables after builtins. This actually bit me, because my first dirty workaround was to simply cast the `columns` argument to `DataFrame` to `list`. Obviously, `list` no longer did what I expected.

For what it's worth, `lst` is also the variable name used at https://github.com/ig-python/ig-markets-api-python-library/blob/master/trading_ig/rest.py#L1038 in the same context.